tests/http/stateless/ApplicationHookTest.php

```
@@ -0,0 +1,183 @@
+            "Overridden 'reinitializeApplication()' hook should be invoked by 'prepareForRequest()'.",
+        );
+    }
+
+    /**
+     * @throws InvalidConfigException if the configuration is invalid or incomplete.
+     */
+    public function testPrepareForRequestCallsOverriddenResetRequestStateHook(): void
+    {
+        $app = $this->applicationRest();
+
+        $response = $app->handle(FactoryHelper::createRequest('GET', 'site/index'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii2\extensions\psrbridge\tests\support\FactoryHelper' in method 'testPrepareForRequestCallsOverriddenResetRequestStateHook'. (undefined)

(StaticAccess)